js-intermediate.test.js

```diff
@@ -3,53 +3,24 @@
 
 // 1. Write the code that would make this test pass.
 
-//describe("multTwo", () => {
-//    test("returns an array with all the numbers multiplied by two", () => {
-//        expect(multTwo([3, 4, 5])).toEqual([6, 8, 10])
-//        expect(multTwo([23, -9, 0])).toEqual([46, -18, 0])
-//        expect(multTwo([4.5, -4.5, 12])).toEqual([9, -9, 24])
- //   })
-//})
-// create function that will multiple the values of array by 2
-//map over array i*2 return value into a new array
+describe("multTwo", () => {
+    test("returns an array with all the numbers multiplied by two", () => {
+        expect(multTwo([3, 4, 5])).toEqual([6, 8, 10])
+        expect(multTwo([23, -9, 0])).toEqual([46, -18, 0])
+        expect(multTwo([4.5, -4.5, 12])).toEqual([9, -9, 24])
+    })
+})
 
-var example = [3, 4, 5]
-var example1 = [23, -9, 0]
-var example2 = [4.5, -4.5, 12]
-
-
-const multTwo = (array) => {
-  return array.map((value) => {
-    return value * 2
-  })
-}
-
-console.log(multTwo(example))
-console.log(multTwo(example1))
-console.log(multTwo(example2))
 
 // 2. Write the code that would make this test pass.
```

Comment:
When coding to make a test pass you don't need to create variable or log your function.